generic/tests/nfs_corrupt.py

```diff
@@ -194,14 +194,17 @@ def check_vm_status(vm, status):
         config.setup()
     except NFSCorruptError as e:
         test.error(str(e))
-    image_name = os.path.join(config.mnt_dir, "nfs_corrupt")
+
+    image_name = "nfs_corrupt"
     params["image_name_stg"] = image_name
```

Comment:
set image_name_stg = nfs_corrupt in cfg, so we can remove line 198&199?

Reply:
Agree with you, I will update it. thanks.